docs/blog/posts/2025-08-07_get-messages-api-guide.md

```
@@ -0,0 +1,213 @@
+      })
+      .catch(error => {
+        console.error(`Error ${error.response?.status}: ${error.response?.data}`);
+      });
+    ```
+
+=== "cURL"
+    ```bash title="Basic Message Retrieval"
+    # Local server example
+    curl -u user:pass "http://localhost:8080/messages?limit=10"
+
+    # Cloud service example with time filtering
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)